autogen.sh

```diff
@@ -47,5 +48,9 @@ set -ex
 $LIBTOOLIZE --copy --force
 aclocal -I m4 -I .
 autoheader
-automake --add-missing --copy --foreign
+AM_OUTPUT=$(automake --add-missing --copy --foreign 2>&1 | egrep -v "(subdir-objects|is in a subdirectory)")
```

Comment:
by moving the automake into a subshell, the `-e` option has no effect.

Reply:
The main issue seems to be that warnings from automake does not return an error, yet it does not complete.

I wonder if deleting configure on top is the easier fix, to avoid the error masking.

Reply:
Sounds simple and clean